tests/foreman/installer/test_installer.py

```diff
@@ -1345,112 +1344,107 @@
+    rpm_version = rpm_version[:-2]
+    assert rpm_version.replace('-', '.') == semodule_version
 
-        'pulp_workers', 'qdrouterd', 'qpidd', 'tomcat'} are started
-        """
-        major_version = get_host_info()[1]
-        services = (
-            'dynflow-sidekiq@orchestrator',
-            'dynflow-sidekiq@worker',
-            'dynflow-sidekiq@worker-hosts-queue',
-            'foreman-proxy',
-            'httpd',
-            'rh-mongodb34-mongod',
-            'postgresql',
-            'pulp_celerybeat',
-            'pulp_resource_manager',
-            'pulp_streamer',
-            'pulp_workers',
-            'puppetserver',
-            'qdrouterd',
-            'qpidd',
-            'rh-redis5-redis',
-            'smart_proxy_dynflow_core',
-            'squid',
-            'tomcat6' if major_version == RHEL_6_MAJOR_VERSION else 'tomcat',
-        )
 
-        # check `services` status using service command
-        if major_version >= RHEL_7_MAJOR_VERSION:
-            status_format = 'systemctl status {0}'
-        else:
-            status_format = 'service {0} status'
+@upgrade
+@tier1
+def test_positive_check_installer_services():
+    """Check if services start correctly
 
-        for service in services:
-            with self.subTest(service):
-                result = ssh.command(status_format.format(service))
-                self.assertEqual(result.return_code, 0)
-                self.assertEqual(len(result.stderr), 0)
+    :id: 85fd4388-6d94-42f5-bed2-24be38e9f104
 
-        # check status reported by hammer ping command
-        result = ssh.command(
-            'hammer -u {0[0]} -p {0[1]} ping'.format(settings.server.get_credentials())
-        )
+    :expectedresults: All services {'elasticsearch', 'foreman-proxy', 'httpd',
 
-        result_output = [
-            service.strip() for service in result.stdout if not re.search(r'message:', service)
-        ]
+    'rh-mongodb34-mongod', 'postgresql', 'pulp_celerybeat', 'pulp_resource_manager',
 
-        # iterate over the lines grouping every 3 lines
-        # example [1, 2, 3, 4, 5, 6] will return [(1, 2, 3), (4, 5, 6)]
-        for service, status, response in zip(*[iter(result_output)] * 3):
-            service = service.replace(':', '').strip()
-            status = status.split(':')[1].strip().lower()
-            response = response.split(':', 1)[1].strip()
-            self.assertEqual(status, 'ok', f'{service} responded with {response}')
+    'pulp_workers', 'qdrouterd', 'qpidd', 'tomcat'} are started
+    """
+    major_version = get_host_info()[1]
+    service_name = 'tomcat6' if major_version == RHEL_6_MAJOR_VERSION else 'tomcat'
+    SATELLITE_SERVICES.add(service_name)
+    for service in SATELLITE_SERVICES:
+        if major_version >= RHEL_7_MAJOR_VERSION:
+            result = ssh.command(f'systemctl status {service}')
+        else:
+            result = ssh.command(f'service {service} status')
```

Comment:
Personally, I liked previous version better. Satellite RHEL version is not going to change during test execution, so it makes sense to extract `if` outside of the loop - we don't waste resources on condition that is the same in every iteration.

It's not going to have visible impact on test execution time, so it's another non-blocking comment.

Reply:
Fixed.